cudamapper/src/main.cpp

```diff
@@ -27,27 +28,27 @@ int main(int argc, char *argv[])
     // TODO: pass kmer and window size as parameters
     std::unique_ptr<claragenomics::IndexGenerator> index_generator = claragenomics::IndexGenerator::create_index_generator(std::string(argv[1]), 15, 15);
     CGA_LOG_INFO("Created index generator");
-    std::cout << "Index generator execution time: " << std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::high_resolution_clock::now() - start_time).count() << "ms" << std::endl;
+    std::cerr << "Index generator execution time: " << std::chrono::duration_cast<std::chrono::milliseconds>(std::chrono::high_resolution_clock::now() - start_time).count() << "ms" << std::endl;
```

Comment:
I was thinking about this, but we don't have a verbose flag yet and if I use `CGA_LOG_INFO` it will only work for Release builds, which is not what we want for this instrumentation code. An alternative is to use `CGA_LOG_ERROR` but this seems wrong since I'm not reporting an error.

Reply:
I agree. Let's keep it this was for now.